packages/vscode-extension/src/handlers.ts

```diff
@@ -160,7 +161,8 @@ export function getSystemInputs():Inputs{
     projectPath: getWorkspacePath(),
     platform: Platform.VSCode,
     vscodeEnv: detectVsCodeEnv(),
-    "function-dotnet-checker-enabled": vscodeAdapter.dotnetCheckerEnabled()
+    "function-dotnet-checker-enabled": vscodeAdapter.dotnetCheckerEnabled(),
+    correlationId: uuid.v4()
```

Comment:
1. how does the correlation Id pass to core?
2. echo user operation has different correlation id, how to handle it?

Reply:
1. pass as property in Inputs object
2. Each operation has unique correlation id